packages/venia-ui/lib/queries/addConfigurableProductsToCart.graphql

```diff
@@ -4,7 +4,7 @@ mutation addConfigurableProductToCart(
     $sku: String!
     $parentSku: String!
 ) {
-    addConfigurableProductsToCart(
+    response: addConfigurableProductsToCart(
```

Comment:
Product add uses different mutations depending on product type. Rename the responses here to simplify the action code for destructuring the response.